pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
```

Comment:
I don't really care what tool we use but we probably want to define our dependencies.  This was fastest for me.